test/e2e/framework/framework.go

```diff
@@ -799,7 +799,7 @@ func (f *Framework) DefaultGameServer(namespace string) *agonesv1.GameServer {
 					Containers: []corev1.Container{{
 						Name:            "game-server",
 						Image:           f.GameServerImage,
-						ImagePullPolicy: corev1.PullIfNotPresent,
+						ImagePullPolicy: corev1.PullAlways,
```

Comment:
Before we close off this PR, let's move this back to `PullIfNotPresent`

Reply:
Done! Added the new image to prod. Also updated the new image in all the files. Could you please review it and merge it?